utils/pom.xml

```diff
@@ -0,0 +1,54 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.redhat.lightblue.migrator</groupId>
+    <artifactId>lightblue-migrator-pom</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+  </parent>
+  <groupId>com.redhat.lightblue.migrator</groupId>
+  <artifactId>lightblue-migrator-utils</artifactId>
+  <packaging>jar</packaging>
+  <version>0.0.1-SNAPSHOT</version>
+  <name>lightblue-migrator: ${project.groupId}|${project.artifactId}</name>
+  <dependencies>
```

Comment:
Define dependencies in root pom's dependencyManagement to move management of all versions except artifacts built by this repo to the root pom.